### PR TITLE
Update to JOL 0.9

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: src/main/java/io/airlift/slice/InstanceSize.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import java.util.stream.Stream;
+
+import static org.openjdk.jol.info.ClassLayout.parseClass;
+
+public final class InstanceSize
+{
+    private InstanceSize() {}
+
+    public static int instanceSize(Class<?>... classes)
```
Comment (findepi) [inline src/main/java/io/airlift/slice/InstanceSize.java]: is there use cases for multiple classes?

Comment (electrum) [reply src/main/java/io/airlift/slice/InstanceSize.java]: We have a few, but probably better to remove this.

**Comment by wendigo** [comment]
JOL 0.10 was released this year with some performance improvements. Maybe it's worth to update @electrum ?

**Comment by dain** [comment]
Superseded by #153

**Pull request closed without merging**